### ISSUE TITLE
Tracking netlib

### ISSUE THREAD

**Issue by konradha** [lfortran/lfortran#763]
We should track progress on how well the libraries on [netlib](https://netlib.org/) can be compiled. Parsing, yes, but actually emitting code / performance comparisons should be looked at regularly.

**Comment by awvwgk** [comment]
Netlib might not be the best address for such libraries as many of them are frozen in time or not always fully standard conformant to begin with (I don't think GFortran will compile all of netlib out of the box). Focusing on a more concrete goal like #512 or #374 seems preferable to not waste resources and time on a big task.

**Comment by konradha** [comment]
You're right. I didn't put it as a high priority item but rather a thing to keep in mind for future reference because eventually, this will be tackled.
